docs/node-termination.md

```diff
@@ -15,4 +15,40 @@ where we compare the creation timestamps of each node.
 
 This method is useful to ensure there are always new nodes in the cluster. If you want to deploy a configuration change
 to your nodes, you can use Escalator to cycle the nodes by terminating the oldest first until all of the nodes are
-using the latest configuration.
\ No newline at end of file
+using the latest configuration.
+
+### Annotating nodes / Stopping termination of selected nodes
+
+For most cases when wanting to exclude a node from termination for maintenance, you should first consider if the Cordon function [**Cordoning of nodes**](./scale-process).
```

Comment:
`you should first consider the [**cordon function**](./scale-process).`

Reply:
Updated